docs/tutorials/developer-journey/level-3/3.5-identities-and-auth.md

```diff
@@ -0,0 +1,288 @@
+
+## Types of identity
+
+First, let's clearly define the difference between the different types of identity and authentication on the Internet Computer. As we've already covered some of these in previous modules, some of this will be a review.
+
+- **Developer identity**: created using `dfx`. It contains a private/public key pair, and has a `principal` data type derived from the public key.  It can be compared to a Bitcoin or Ethereum wallet address.
+
+- **Principals**: generic identifiers that are used for users, canisters, and potentially other future concepts. There is intentionally no way to tell canister principal IDs and user principal IDs apart. You can learn more in the IC [specification](https://internetcomputer.org/docs/current/references/ic-interface-spec#principal).
+
+- **Principal identifiers**: principals associated with a user principal ID. Principal identifiers are related to account identifiers, but use a different format. Each principal identity can control multiple accounts in the ICP (and other) ledgers.
+
+- **Account identifier**: the identifier associated with your ICP ledger account, as specified in the [ledger](../../../developer-docs/integrations/ledger/index.md) specification. 
+
+- **Wallets**: used to store forms of currency or other assets, such as cycles, ICP, or NFTs. Developers primarily use a cycles wallet to send cycles to and from canisters. 
+
+- **Internet Identity**: the IC's native authentication service. Internet Identity doesn't use usernames and passwords; instead it uses a **passkey** that is stored in your local device's hardware.
+
+In this guide, we'll primarily look at how to use Internet Identity, as the other forms of identity have been covered in previous modules.
+
+## Internet Identity
+
+On traditional, Web2 websites and applications, many of them have the option for you to 'Sign in with Google', which allows you to authenticate with the application using your username and password credentials for Google, preventing you from having to make a new username and password for that application.
+
+On the Internet Computer, a native form of identity is used for authentication, called the Internet Identity.
+
+Internet Identity is a form of the WebAuthn API that allows users to register and authenticate with websites or applications without using a username and password. Instead, users can use a **passkey**, which is a unique cryptographic public/private key pair that is stored in the secure hardware chip of your computer or phone. This allows you to authenticate your Internet Identity using methods that unlock your device, such as TouchID, FaceID, or another method. 
```

Comment:
could we clarify this? is it accurate to state that "Internet Identity is an implementation of the WebAuthn API"?